reactDashboard/src/components/Footer/Footer.jsx

```diff
@@ -147,7 +150,7 @@ const Footer = () => {
               <FontAwesomeIcon
                 icon={faLinkedinIn}
                 size="2x"
-                color="#133a53"
+                color={darkmode ? DARK_SOCIAL_LOGO : SOCIAL_LOGO}
```

Comment:
Is it possible for these FA logos to use css - the class name third-party-logo is already there? If so you could move these colour declarations into the css and not have to pass the property down into the jsx.